Formula/pysvn.rb

```diff
@@ -0,0 +1,70 @@
+class Pysvn < Formula
+  desc "Python interface to Subversion"
+  homepage "http://pysvn.tigris.org/"
+  url "http://pysvn.barrys-emacs.org/source_kits/pysvn-1.8.0.tar.gz"
+  sha256 "39596f4884ed689cdb5a4e210e421724302a566c7ba756cc4d46bbfeb0c8326b"
+
+  depends_on "openssl"
+  depends_on "subversion"
+  depends_on :python => :recommended
+  depends_on :python3 => :recommended
+
+  # Fix configure to respect CPPFLAGS, CXXFLAGS and LDFLAGS
+  patch :DATA
+
+  def install
+    if build.without?("python") && build.without?("python3")
+      odie "pysvn: at least one of --with-python or --with-python3 must be specified"
+    end
+    cd "Source" do
+      inreplace "setup_configure.py", "%(PYTHON_FRAMEWORK)s", "-undefined dynamic_lookup"
```

Comment:
Please report this upstream as well. We consider it an issue that upstreams can & should be aware of: http://blog.tim-smith.us/2015/09/python-extension-modules-os-x/